persistence/rocksdb/src/main/java/org/infinispan/persistence/rocksdb/RocksDBStore.java

```diff
@@ -78,6 +77,9 @@
    private static final Log log = LogFactory.getLog(MethodHandles.lookup().lookupClass(), Log.class);
    private static final boolean trace = log.isTraceEnabled();
 
+   private static final byte[] BEGIN_KEY = new byte[0];
+   private static final byte[] END_KEY = new byte[] {(byte) 0xffffffff};
```

Comment:
This is equivalent to `new byte[] {(byte) 0xff}`, it creates an array with a single byte element.